src/delay.ts

```diff
@@ -12,7 +12,7 @@ export async function delay<T>(delayTimeMs: number): Promise<void>;
 export async function delay<T>(delayTime: any, value?: T): Promise<void | T> {
   return new Promise(
     // tslint:disable-next-line:no-any (typed by overload signatures)
-    resolve => setTimeout(() => resolve(value), delayTime),
+    (resolve) => setTimeout(() => resolve(value), delayTime),
```

Comment:
personally hate this lint change, but if you feel strongly this is the right strategy I don't really care as long as it's consistent and is handled automatically with the --fix operation